MovieHub/src/styles/NavigationBar.css

```
@@ -802,15 +802,23 @@
   border: 1px solid rgba(255,255,255,0.06);
   background: rgba(255,255,255,0.02);
   color: #fff;
   outline: none;
+  box-sizing: border-box;
+  resize: none;
+  max-height: 240px;
+  min-height: 40px;
+  overflow-y: hidden;
+  line-height: 1.3;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.2.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)